projects/plugins/crm/includes/ZeroBSCRM.DAL3.Helpers.php

```diff
@@ -3112,13 +3112,6 @@ function zeroBS_buildObjArr($arraySource=array(),$startingArray=array(),$fieldPr
 		                        $retArray[$outputPrefix.$fK] = intval($retArray[$outputPrefix.$fK]);
 		                        break;
 
-
-		                    case 'textarea':
-
-		                        $retArray[$outputPrefix.$fK] = zeroBSCRM_textProcess($arraySource[$fieldPrefix.$fK]);
```

Comment:
Perhaps we should look into a way to still sanitize the value instead of skipping it completely?

What if we use `sanitize_textarea_field()` instead?

Reply:
line `3153`

Reply:
We are sanitizing the value by the default value of the `switch` block with the `sanitize_text_field` function, line `3153`. But we could use `sanitize_textarea_field` to avoid problem with new lines `\n`

Reply:
@kallehauge done!